packages/components/src/local/mapping/helpers/cell-types.ts

```diff
@@ -0,0 +1,17 @@
+import { SergeHex } from '@serge/custom-types'
```

Comment:
This isn't a technical shortcoming, but having two files titled `cell-types.ts` feels scary.

Could we rename the helper function (& filename) to `cellTypeFor`?

Aah, I think I've also spotted a potential performance improvement.  I may be wrong, but it looks like we read/parse `cellTypesList` lots and lots of times.  But, it doesn't change.

Is it an option to load the data object once when `create-grid` runs, then pass the structure in to this function as a parameter?

Reply:
Yes inded those could be improved

Reply:
I changed the name and extracted the list in create-grid